### DISCUSSION TITLE
¿Cómo es el procedimiento para activar KB en una compañía?

### DISCUSSION THREAD

**Discussion by seiturra** [Cotalker/documentation#91] [2. Questions and Answers] [answered]
Hola a todos,

Necesito activar KB para un cliente y quisiera saber cómo es el conducto regular para hacerlo.

Saludos.

**Comment by acedward** [comment, marked as answer]
Para activar el KB-Legacy se puede realizar por cualquier persona quien requieren permisos internos:
1. Debe crearse la base de datos directamente en la máquina de KB. Por favor contactame por interno, con la compañía y entornos en particuar en donde se debe activar. 
2. Existe un repositorio privado donde se encuentran todos los comandos disponibles una vez activado: https://github.com/Moblox/knowledgebase-api

**Reply by jdurandpaz** [reply]
@SeIturra ?

**Reply by seiturra** [reply]
@jdurandpaz necesito un poco más de información que estoy consultando en el canal del proyecto.

**Reply by acedward** [reply]
No hay más feedback. Voy a cerrarlo.
